docs/usage.md

```diff
@@ -365,12 +371,26 @@ The [Nextflow DSL2](https://www.nextflow.io/docs/latest/dsl2.html) implementatio
 
         ```nextflow
         process {
-            withName: PANGOLIN {
-                conda = 'bioconda::pangolin=3.0.5'
+            withName: 'NEXTCLADE_DATASETGET|NEXTCLADE_RUN' {
+                conda = 'bioconda::nextclade=1.10.1'
             }
         }
         ```
 
+##### Nextclade datasets
+
+A [`nextclade dataset`](https://docs.nextstrain.org/projects/nextclade/en/latest/user/datasets.html#nextclade-datasets) feature was introduced in [Nextclade CLI v1.3.0](https://github.com/nextstrain/nextclade/releases/tag/1.3.0) that fetches input genome files such as reference sequences and trees from a central dataset repository. We have uploaded Nextclade dataset [v2022-01-18](https://github.com/nextstrain/nextclade_data/releases/tag/2022-01-24--21-27-29--UTC) to [nf-core/test-datasets](https://github.com/nf-core/test-datasets/blob/viralrecon/genome/MN908947.3/nextclade_sars-cov-2_MN908947_2022-01-18T12_00_00Z.tar.gz?raw=true), and for reproducibility, this will be used by default if you specify `--genome 'MN908947.3'` when running the pipeline. However, there are a number of ways you can use a more recent version of the dataset:
+
+* Supply your own by setting: `--nextclade_dataset <PATH_TO_DATASET>`
+* Let the pipeline create and use the latest version by setting: `--nextclade_dataset false --nextclade_dataset_tag false`
```

Comment:
I'd suggest not including this as an option, as it allows for non-reproducibility

Reply:
We have had a number of users that just want to use the latest version and this allows them to do that 🤷🏽

Reply:
🤦